products/sle/main.pm

```diff
@@ -756,6 +756,9 @@ sub load_consoletests() {
             loadtest "console/zypper_lifecycle";
         }
         loadtest 'console/install_all_from_repository' if get_var('INSTALL_ALL_REPO');
+        if (check_var("SCC_REGISTER", "installation") && check_var("DESKTOP", "gnome")) {
```

Comment:
@mimi1vx this is where I suggested to check for SCC_ADDONS and PATTERNS instead of SCC_REGISTER. The reason for PATTERNS is that the new module - at least for now - should only be called in the one scenario where we check module registration (SCC_ADDONS) *and* installation (PATTERNS) from it. I don't see a purpose right now to test this module in more scenarios where we register modules for specific causes, e.g. register the toolchain module only to test the toolchain. No need to test "module deregistration" there IMHO. Would only waste time and potentially break more, e.g. when only one module is registered.